extensions/azurecore/src/azurecore.d.ts

```diff
@@ -68,6 +68,10 @@ declare module 'azurecore' {
 	export interface IExtension {
 		getSubscriptions(account?: azdata.Account, ignoreErrors?: boolean, selectedOnly?: boolean): Thenable<GetSubscriptionsResult>;
 		getResourceGroups(account?: azdata.Account, subscription?: azureResource.AzureResourceSubscription, ignoreErrors?: boolean): Thenable<GetResourceGroupsResult>;
+		getSqlManagedInstances(account: azdata.Account, subscriptions: azureResource.AzureResourceSubscription[], ignoreErrors?: boolean): Thenable<GetSqlManagedInstancesResult>;
```

Comment:
These don't need to be Thenables, just use Promise (feel free to update the others as well or leave them, it's fine either way. But Thenable is unnecessary and just being more restrictive than we need to be)